packages/web-pixels-extension/src/types/standard-events-V0/index.ts

```diff
@@ -0,0 +1,59 @@
+// Code generated by jtd-codegen for TypeScript v0.2.1
+
+export type StandardEventsV0 =
+  | StandardEventsV0CheckoutCompleted
+  | StandardEventsV0CheckoutStarted
+  | StandardEventsV0CollectionView
+  | StandardEventsV0PageView
+  | StandardEventsV0PaymentInfoSubmitted
+  | StandardEventsV0PerformedSearch
+  | StandardEventsV0ProductAddedToCart
+  | StandardEventsV0ProductView;
+
+export interface StandardEventsV0CheckoutCompleted {
+  event_name: 'checkout_completed';
+  event_schema_version: string;
+  page_url: string;
+}
+
+export interface StandardEventsV0CheckoutStarted {
+  event_name: 'checkout_started';
+  event_schema_version: string;
+  page_url: string;
+}
+
+export interface StandardEventsV0CollectionView {
+  event_name: 'collection_view';
+  event_schema_version: string;
+  page_url: string;
```

Comment:
It's unfortunate we aren't making these into more JS-friendly identifiers

Reply:
CamelCase as opposed to snake_case

Reply:
we picked camel_case to follow Google Analytics' convention (along with many other partners like Facebook): https://developers.google.com/analytics/devguides/collection/ga4/ecommerce?client_type=gtag#view_item_list

Reply:
Yeah, I think we opted for snake_case to have a consistent event object between client and server. Are we still in favour of keeping consistent?